### PR TITLE
Cleanup generators

### PR DESCRIPTION
**Pull request by nicolas-brousse**
- Move all generators in `Dekorator` namespace. This allow users to overwrite templates of generators in a Rails application.
- Adds `app/decorators/concerns/.keep` on installation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ This command will create an `ApplicationDecorator` file.
 
 Generate a new decorator with the `decorator` generator:
 
-    $ rails generate decorator user
+    $ rails generate dekorator:decorator user
```
Comment (Spone) [inline README.md]: I'm not a fan of this long syntax... In Komponent we use `rails generate component foobar`, not `rails generate komponent:component foobar`. There is no other way?

Comment (nicolas-brousse) [reply README.md]: Yes I know, it's why it was like this at first.

Pundit, ActionPolicy do it with gem name as namespace. I searched but didn't found a way to do some alias. And didn't found any gem who do like we do in komponent :/

Comment (Spone) [reply README.md]: Draper does: https://github.com/drapergem/draper#generators

Comment (nicolas-brousse) [reply README.md]: I did a second generator that invoke the first one. Don't know if it's a good think.

**Comment by nicolas-brousse** [comment]
Finally I moved back `rspec` and `unit_test` generators, otherwise they seems to be not invokable. I don't really know why. So let keep them like they were.

**Pull request merged**